docs/src/cell.md

```diff
@@ -30,7 +32,7 @@ Description: The Cell object represents a biological cell within a simulation. I
 
 - void setImmortal(bool isImmortal): sets the global variable immortal to the given value
 
-- void _ready() override: runs once when the Cell has been created and is ready to be used; points _cellState to this Cell's CellState Node
+- void _ready() override: runs once when the Cell has been created and is ready to be used; points _cellState to this Cell's CellState Node; sets homeostasis and reproduction costs
```

Comment:
One thing to note is that `_ready()` happens when an object is added to the node tree (using `add_child()` for example), not when it's created.